src/app/view/templates/beol/beol.component.ts

```diff
@@ -243,7 +243,7 @@ export class BeolComponent implements OnInit {
 
         const Johann_Daniel_Schoepflin = new Correspondent("Johann Daniel Schoepflin", "(DE-588)118610260");
 
-        const Willem_Jacob_SGravesande = new Correspondent("Willem Jacob SGravesande", "(DE-588)118939297");
+        const Willem_Jacob_SGravesande = new Correspondent("Willem Jacob sGravesande", "(DE-588)118939297");
```

Comment:
name should be Willem Jacob Gravesand

Reply:
sGravesand is a variant: http://d-nb.info/gnd/118939297

in BEBB, the person appears under "S"

Reply:
let's leave it like this then :-)